rasa/nlu/selectors/response_selector.py

```diff
@@ -313,20 +322,22 @@ def preprocess_train_data(self, training_data: TrainingData) -> RasaModelData:
         self.index_label_id_mapping = self._invert_mapping(label_id_index_mapping)
 
         self._label_data = self._create_label_data(
-            training_data, label_id_index_mapping, attribute=RESPONSE
+            training_data, label_id_index_mapping, attribute=label_attribute
         )
 
         model_data = self._create_model_data(
             training_data.intent_examples,
             label_id_index_mapping,
-            label_attribute=RESPONSE,
+            label_attribute=label_attribute,
         )
 
         self._check_input_dimension_consistency(model_data)
 
         return model_data
 
-    def _full_response(self, label: Dict[Text, Any]) -> Optional[Dict[Text, Any]]:
+    def _full_response(
+        self, label: Dict[Text, Any]
+    ) -> Tuple[Any, List[Dict[Text, Any]]]:
```

Comment:
not sure, but I think we can replace `Tuple[Any, List[Dict[Text, Any]]]` with `Tuple[Text, List[Dict[Text, Any]]]`

Reply:
I thought so too, but type check fails I think bc `label` as type `Dict[Text, Any]` which is valid because a value in that dict can be `None` as well.

Reply:
Can we then change the type of `label` to `Dict[Text, Optional[Text]]`?